src/skmatter/_selection.py

```diff
@@ -57,19 +50,20 @@ class GreedySelector(SelectorMixin, MetaEstimatorMixin, BaseEstimator):
 
     score_threshold : float, default=None
         Threshold for the score. If `None` selection will continue until the
-        n_to_select is chosen. Otherwise will stop when the score falls below the threshold.
-        Stored in :py:attr:`self.score_threshold`.
+        n_to_select is chosen. Otherwise will stop when the score falls below the
+        threshold. Stored in :py:attr:`self.score_threshold`.
```

Comment:
```suggestion
        n_to_select is chosen. Otherwise will stop when the score falls below the  threshold. 
        Stored in :py:attr:`self.score_threshold`.
```

Reply:
Same here 88 characters per line.